doc/python/sphinx/glossary.rst

```diff
@@ -27,3 +27,17 @@ Glossary
     the returned object doesn't have restrictions (i.e. ``None`` allowed). The
     deserializer is invoked with inbound message bytes on both the server side
     and the client-side.
+
+  wait_for_ready
+    If an RPC is issued but the channel is in TRANSIENT_FAILURE or SHUTDOWN
+    states, the RPC is unable to be transmitted promptly. By default, gRPC
+    implementations SHOULD fail such RPCs immediately. This is known as "fail
+    fast," but the usage of the term is historical. RPCs SHOULD NOT fail as a
+    result of the channel being in other states (CONNECTING, READY, or IDLE).
+
+    gRPC implementations MAY provide a per-RPC option to not fail RPCs as a
```

Comment:
Suggestion to remove the RFC lingo: "When the `wait_for_ready` option is specified, the library will queue RPCs until the channel is READY. Any submitted RPCs may still fail before the READY state is reached for other reasons, e.g. the client channel has been shut down or the RPC's deadline has been reached."

Reply:
👍  Your version is better.